Makefile

```diff
@@ -1,19 +1,51 @@
 DOT := $(shell command -v dot 2> /dev/null)
 
-up:
+rm-up:
 	docker network inspect ssdcrmdockerdev_default >/dev/null || docker network create ssdcrmdockerdev_default
 	docker compose -f dev.yml -f rm-services.yml up -d ${SERVICE} ;
 	pipenv install --dev
 	./setup_pubsub.sh
+
+rh-up:
+	docker network inspect ssdcrmdockerdev_default >/dev/null || docker network create ssdcrmdockerdev_default
+	docker compose -f rh-dev.yml -f rh-services.yml up -d ${SERVICE} ;
+	pipenv install --dev
+	./setup_pubsub.sh
+
+rmrh-up:
+	docker network inspect ssdcrmdockerdev_default >/dev/null || docker network create ssdcrmdockerdev_default
+	docker compose -f dev.yml -f rm-services.yml -f rh-dev.yml -f rh-services.yml up -d ${SERVICE} ;
+	pipenv install --dev
+	./setup_pubsub.sh
 	
-down:
```

Comment:
Can we preserve `make down` as a command to stop everything? There's no harm passing docker compose down extra docker compose files, even if they haven't been started.